core/src/mindustry/world/Tiles.java

```diff
@@ -7,18 +7,40 @@
 
 import java.util.*;
 
-/** A tile container. */
+/**
+ * Contains a list of tiles with extended behavior
```

Comment:
every class should use single-line doc comments whenever possible, multiple blank lines waste a lot of space and don't make anything clearer